cloudimg/common.py

```diff
@@ -25,19 +24,15 @@ class PublishingMetadata(object):
                                    the image
         groups (list, optional): Groups which will have permission to use the
                                  image
-        snapshot_account_ids (list, optional): Accounts which will have
-                                            permission to create the snapshot
+        tags (dict, optional): Tags to be applied to the image.
     """
 
     def __init__(self, image_path, image_name, description=None,
```

Comment:
Again this change seems to be backwards-incompatible, if anyone was creating a PublishingMetadata instance before while providing a snapshot_name or snapshot_account_ids, their code will break.

Can the backwards-incompatible changes please be avoided?

Reply:
Ah, maybe it was because of the commit on top of it which removed the `snapshot_accout_ids` from this position right?

After dropping the second commit it seems to be ok, can you please confirm?

Reply:
Yes, I think this was fixed when the PR was revised.